test/integration/features/basic.feature

```diff
@@ -43,10 +43,8 @@ Feature: Basic test
         And stdout should contain "Setting up KVM"
         And stdout should contain "Installing libvirt service and dependencies" 
         And stdout should contain "Adding user to libvirt group"
-        And stdout should contain "Will use root access: add user to libvirt group"
         And stdout should contain "Enabling libvirt"
```

Comment:
I'm a bit surprised this does not show in the output if this is the first run of crc setup on a newly installed machine.